src/lib/components/sidebar/Sidebar.tsx

```diff
@@ -19,44 +19,73 @@ const Sidebar: React.FunctionComponent = () => {
   const SidebarData = [
     {
       name: 'home',
+<<<<<<< HEAD:src/components/sidebar/Sidebar.tsx
```

Comment:
Please run through your code, and remove those characters. I think you messed up resolving the merge conflicts.